spec/features/studios/index_spec.rb

```diff
@@ -0,0 +1,35 @@
+require "rails_helper"
+
+RSpec.describe Studio do
```

Comment:
Because we are creating tests for a view page and not for a model, this first `describe` line is actually going to be a little different. What you have here would work if this were a model test and we were testing the `Studio` model. Because we are testing a view, however, we want to pass in a string. This string can be anything you want but should make some sense.

```ruby
RSpec.describe "Studio index page" do
```